core/build.gradle

```diff
@@ -2,13 +2,14 @@ description = 'gRPC: Core'
 
 dependencies {
     compile project(':grpc-context'),
+            libraries.flogger,
+            libraries.flogger_backend,
```

Comment:
This is a classpath dep needed to work, but we don't reference any classes from it.